### DISCUSSION TITLE
Instructions on how to run?

### DISCUSSION THREAD

**Discussion by Ryanmt2** [HackTJ/live#58] [Q&A] [answered]
I am looking to have an experiment with some applications with the idea of using them at a hackathon in a months time - This project sounds interesting but I am struggling to get it going with any of the docker compose files included. I have spun up docker-compose.yml and with a bit of tweaking got it to start-up but I am not seeing anything served on port 8000.

Are there any more complete instructions on how to run locally/in prod anywhere?

**Comment by sumanthratna** [comment, marked as answer]
Hi @Ryanmt2! Thanks for checking out HackTJ Live. 

For docs on setting up a local server, you'll want to read https://github.com/HackTJ/live/blob/main/docs/development.md. In short, run `./start_live dev` and then open `localhost:8000`

For deploying a production instance, take a look at https://github.com/HackTJ/live/blob/main/docs/deployment/self-hosted.md. In short, run `./start_live prod` and open `localhost:8000`. We recommend setting up NGINX or some other load balancer in front of port 8000. We also have a basic Heroku setup supported (to scale well you'll probably need to spend some money on a better Heroku plan): https://github.com/HackTJ/live/blob/main/docs/deployment/heroku.md

Let me know if you have any issues or other questions!
